### PR TITLE
Updated pysat branch used by Travis

### PR DESCRIPTION
**Pull request by aburrell**
Updated Travis CI to use 3.0.0 RC branch of pysat.  Addresses https://github.com/pysat/pysat/pull/676, and may be updated further once the pysat PR is closed.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,7 +36,7 @@ install:
   - pip install pysatCDF >/dev/null
   # Custom pysat install
   - cd ..
-  - git clone --single-branch --branch develop-3 https://github.com/pysat/pysat.git
+  - git clone --single-branch --branch v3-0-rc1 https://github.com/pysat/pysat.git
```
Comment (rstoneback) [inline .travis.yml:39]:
```suggestion
  - git clone --single-branch --branch main https://github.com/pysat/pysat.git
```

File: .travis.yml [hunk 2]
```diff
@@ -36,7 +36,7 @@ install:
   - pip install pysatCDF >/dev/null
```
Comment (jklenzing) [inline .travis.yml:36]:
```suggestion
```

**Comment by jklenzing** [comment]
Looks like we need to merge this in for the updates to test_instruments.py before other work can be done.  Should we update to Github Actions here or save that for a separate pull? @aburrell?

**Comment by aburrell** [comment]
Either way is ok for me.  Let's just not remove the doc tests from travis until they are also in GitHub actions.

**Comment by jklenzing** [comment]
@rstoneback, since this is holding up your branches, it's up to you.

**Comment by rstoneback** [comment]
I'm fine either way. I figured out there is still a bit more work on incorporating version into the filename to ensure there are no mixed data loads. Plus I'm guessing the xarray transition (separate pull) will take some time. I don't know how long the testing transition will take but I think I'll take longer :)

**Comment by jklenzing** [comment]
I've gone ahead and opened #20 as a replacement branch for this one.  No docs yet, so documentation testing is not run here.  However, a candidate workflow for doc testing is available in pysat/pysatSpaceWeather#47.

**Comment by jklenzing** [comment]
Closing since #20 is merged

**Pull request closed without merging**